src/components/select/select.vue

```diff
@@ -3,7 +3,7 @@
         <div
             :class="selectionCls"
             ref="reference"
-            @click="toggleMenu">
```

Comment:
This change will make `toggleMenu` method obsolete. In this case you should also remove it.